src/System Application/App/Guided Experience/src/Guided Experience/GuidedExperienceImpl.Codeunit.al

```diff
@@ -569,6 +571,44 @@ codeunit 1991 "Guided Experience Impl."
             InsertGuidedExperienceItemsInTempVar(GuidedExperienceItem, TempGuidedExperienceItem);
     end;
 
+    procedure LoadTopBanners(var TenantMediaSetStandard: Record "Tenant Media Set"; var TenantMediaSetCompleted: Record "Tenant Media Set"; var TenantMediaSetInfo: Record "Tenant Media Set"): Boolean
+    begin
+        if not LoadFileFromTenantMediaSet(TenantMediaSetStandard, 'AssistedSetup-NoText-400px.png') then
+            InsertSystemFileToTenantMediaSet(TenantMediaSetStandard, 'images/', 'AssistedSetup-NoText-400px.png');
+
+        if not LoadFileFromTenantMediaSet(TenantMediaSetCompleted, 'AssistedSetupDone-NoText-400px.png') then
+            InsertSystemFileToTenantMediaSet(TenantMediaSetCompleted, 'images/', 'AssistedSetupDone-NoText-400px.png');
+
+        if not LoadFileFromTenantMediaSet(TenantMediaSetInfo, 'AssistedSetupInfo-NoText.png') then
+            InsertSystemFileToTenantMediaSet(TenantMediaSetInfo, 'images/', 'AssistedSetupInfo-NoText.png');
+
+        exit((TenantMediaSetStandard."Media ID".HasValue) and (TenantMediaSetCompleted."Media ID".HasValue) and (TenantMediaSetInfo."Media ID".HasValue));
+    end;
+
+    procedure LoadFileFromTenantMediaSet(var TenantMediaSet: Record "Tenant Media Set"; FileName: Text[250]): Boolean
+    var
+        TenantMedia: Record "Tenant Media";
+    begin
+        TenantMedia.SetFilter(Description, FileName);
+        if TenantMedia.FindFirst() then
+            if TenantMediaSet.Get(TenantMedia.ID, TenantMedia.ID) then
+                exit(TenantMediaSet."Media ID".HasValue);
+    end;
+
+    procedure InsertSystemFileToTenantMediaSet(var TenantMediaSet: Record "Tenant Media Set"; FilePath: Text[100]; FileName: Text[250])
```

Comment:
I think this procedure should be set to local. I would like a function that first looks in the tenant media, if there is nothing there, then it should be inserted from the resources. 
And then the FilePath is also useless, it is currently always "images/".